abdulfataiaka/pluralsight-admin
# #161341545 Implement author administration

#### What does this PR do
- Adds implementations for authors administration

#### Description of work done
- Add feature to view all authors
- Add feature to delete an author

#### How should it be manually tested
- Clone the repository `git clone https://github.com/abdulfataiaka/pluralsight-admin.git`
- Checkout to this branch
- Ensure to see an `Authors` link on the header
- Visit authors page to see all author
- Ensure you can delete an author without course 

#### Relevant pivotal tracker story
[161284348](https://www.pivotaltracker.com/story/show/161284348)

#### Screenshots
<img width="1440" alt="screen shot 2018-10-19 at 3 26 38 pm" src="https://user-images.githubusercontent.com/32123313/47224444-71674480-d3b3-11e8-82d0-9e4200761270.png">
<img width="1440" alt="screen shot 2018-10-19 at 3 26 40 pm" src="https://user-images.githubusercontent.com/32123313/47224445-71674480-d3b3-11e8-981e-9ea76d4c799c.png">
<img width="1440" alt="screen shot 2018-10-19 at 3 26 47 pm" src="https://user-images.githubusercontent.com/32123313/47224446-71ffdb00-d3b3-11e8-8c49-457a15da88e8.png">
